packages/datadog-instrumentations/src/helpers/hook.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const path = require('path')
+const iitm = require('../../../dd-trace/src/iitm')
+const ritm = require('../../../dd-trace/src/ritm')
+
+function Hook (modules, onrequire) {
+  if (!(this instanceof Hook)) return new Hook(modules, onrequire)
```

Comment:
This function is never actually called with `new`. If we assume that, then we can do away with this line and make all the properties on `this` into local variables.

Reply:
So just a function that returns a plain object with `unhook` on it?

Reply:
I forgot that we sill need unhook. Disregard for now.